apps/portals/admin/src/auth.ts

```diff
@@ -23,6 +23,8 @@ if (userMocked) {
       'profile',
       AdminPortalScope.delegations,
       AdminPortalScope.airDiscountScheme,
+      AdminPortalScope.regulationAdmin,
+      AdminPortalScope.regulationAdminManage,
```

Comment:
Do they need two "roles" for this admin?

Reply:
Yes, good question, we actually do need two roles.

* One is to create new regulations, and can not see other regulations. (`AdminPortalScope.regulationAdmin`)
* The other can create, manage, see all regulations and publish regulations. (`AdminPortalScope.regulationAdminManage`)